web/wp-content/themes/dctx-v2/inc/acf.php

```diff
@@ -0,0 +1,449 @@
+<?php
+/**
+ * Custom ACF functions.
+ *
+ * A place to custom functionality related to Advanced Custom Fields.
+ *
+ * @package dctx
+ */
+
+// If ACF isn't activated, then bail.
+if ( ! class_exists( 'acf' ) ) {
+	return false;
+}
+
+/**
+ * Associate the possible block options with the appropriate section.
+ *
+ * @author WDS
+ * @param  array $args Possible arguments.
+ */
+function dctx_display_block_options( $args = array() ) {
+
+	// Get block background options.
+	$background_options = get_sub_field( 'background_options' ) ? get_sub_field( 'background_options' ) : get_field( 'background_options' )['background_options'];
+
+	// Get block other options.
+	$other_options = array();
+
+	// Set our Other Options if we have them. Some blocks may not.
+	if ( get_sub_field( 'other_options' ) ) {
+		$other_options = get_sub_field( 'other_options' );
+	} elseif ( get_field( 'other_options' ) ) {
+		$other_options = get_field( 'other_options' )['other_options'];
+	}
+
+	// Get block display options.
+	$display_options = array();
+
+	// Set our Display Options if we have them. Some blocks may not.
+	if ( get_sub_field( 'display_options' ) ) {
+		$display_options = get_sub_field( 'display_options' );
+	} elseif ( get_field( 'display_options' ) ) {
+		$display_options = get_field( 'display_options' )['display_options'];
+	}
+
+	// Get the block ID.
+	$block_id = dctx_get_block_id( $args['block'] );
+
+	// Setup defaults.
+	$defaults = array(
+		'background_type' => $background_options['background_type']['value'],
+		'container'       => 'section',
+		'class'           => 'content-block',
+		'font_color'      => $display_options['font_color'],
+		'id'              => $block_id,
+	);
+
+	// Parse args.
+	$args = wp_parse_args( $args, $defaults );
+
+	$background_video_markup = $background_image_markup = '';
```

Comment:
:no_entry_sign: **Error**: Assignments must be the first block of code on a line (*Squiz.PHP.DisallowMultipleAssignments.Found*).